### ISSUE TITLE
Remove email from member response

### ISSUE THREAD

**Issue by spaenleh** [graasp/graasp#718]

**Comment by pyphilia** [comment]
In fact it's not that easy. I think we still need emails to show them in the share modal. But it's true we should fetch them only on admin permissions. 
Probably this means we need to refactor a bit the memberships. I can think of 3 use cases:
- knowing what permission (and which root) you have for an item
- fetching all memberships to show who has access to the item (read/public)
-  fetching all memberships to manage them (admin)

Probably we should have 2 endpoints, one for admin that returns emails and the second for all use without email. What do you think?

What do you think?

**Comment by spaenleh** [comment]
I agree on the fact that we would need to display the emails in the memberships table. For all other operations I think we can do without the email. So by default I would not return the email in the member repsonse, instead I would modify the membership response schema to explicitly return the email (on admin access), so maybe this could be a "getCompleteMemberships"  call that would be different from the default "getMemberships" to just check who has access to what ?

**Comment by spaenleh** [comment]
What is the status on this ? @pyphilia 

I think we made some progress, are we there yet ?
